### ISSUE TITLE
test/markdown: flaky test

### ISSUE THREAD

**Issue by LiliC** [operator-framework/operator-sdk#1103]
Have run into this flaky test few times:
```
19.12s$ make test/markdown
./hack/ci/marker --root=doc
HTTP failure           (pool.sks-keyservers.net -> http://pool.sks-keyservers.net/) at doc/dev/release.md:80 : failed to lookup address information: Name or service not known
HTTP failure           (SKS PGP Public Key Server -> http://pool.sks-keyservers.net/) at doc/dev/release.md:90 : failed to lookup address information: Name or service not known
make: *** [test/markdown] Error 1
The command "make test/markdown" exited with 2.
```
and

```
21.62s$ make test/markdown
./hack/ci/marker --root=doc
Found broken url       (pool.sks-keyservers.net -> http://pool.sks-keyservers.net/) at doc/dev/release.md:80 : 502 Bad Gateway
Found broken url       (SKS PGP Public Key Server -> http://pool.sks-keyservers.net/) at doc/dev/release.md:90 : 502 Bad Gateway
make: *** [test/markdown] Error 1
The command "make test/markdown" exited with 2.
```

Tried`http://pool.sks-keyservers.net` and it resolves for me.

cc @AlexNPavel

**Comment by AlexNPavel** [comment]
Yeah, I'm not sure why it is doing this. It seems like some keyservers have been having weird 502 issues. This started happening after we changed the URL from pgp.mit.edu to the sks pool, since the mit server is unreliable and wasn't working when we were trying to add Ish's public key to the pool. I think we should wait maybe about 1 week, and if it continues just remove the hyperlink.

**Comment by LiliC** [comment]
> I think we should wait maybe about 1 week, and if it continues just remove the hyperlink.

That sounds good 👍

**Comment by hasbro17** [comment]
Should be fixed via #1108
